bin/GameIndex.yaml

```diff
@@ -34215,6 +34309,14 @@ SLUS-20546:
     - OPHFlagHack # Avoid hanging at the "press start" screen.
   # Comments from old GameIndex.dbf for this Game
   #  Needs EmotionEngine set to "Interpreter" and "Enable EE Cache" enabled.
+  patches:
```

Comment:
OPH flag hack need to be removed here to make any use of new patches. Also comment about disabling automatic gamefixes is not needed then in gamedb (and doesn't make sense with added patches).